test/end2end/dartdoc-0.20.0.json

```diff
@@ -1322,6 +1322,17 @@
             "amount": 1608,
             "fraction": 4000
           }
+        },
+        {
+          "code": "dartfmt.warning",
+          "level": "hint",
+          "title": "Format `lib/src/model.dart`.",
+          "description": "Run `dartfmt` to format `lib/src/model.dart`.",
```

Comment:
Do we tell people to 'flutter format' if flutter?

Reply:
Yes.